### PR TITLE
Update faq.ts

### PR DESCRIPTION
**Pull request by amilkh**
also please add hyperlinks to Privacy Policy and Cookie Policy wherever mentioned

### CODE REVIEW

**Comment by gevuong** [comment]
@a-wong-8 do you mind reviewing this?

I contributed the following:

- [ ] updated text in ai page with Amil's proposed changes.
- [ ] replaced text with hyperlinks in faq page and "Do Not Sell My Personal Information" modal. This required restructuring the metadata to support `link` and `text` types.
- [ ] fixed bug where an extra `/` was added to href in story page.
- [ ] Removed the following warning. The warning was significantly longer in the browser console.
Screenshot 2024-10-01 at 3 56 26 PM

File: app/i18n/en/footer.ts [hunk 1]
```diff
@@ -4,86 +4,170 @@ const footer: Footer = {
...
     },
   ],
   sectionPrivacyPolicy: [
     {
       question: 'Argentina',
-      answer: 'Política de Privacidad',
-      link: 'https://drive.google.com/file/d/1Fef7l77QP3EZboej6mw_q7fBNrayvs03/view',
+      answer: [
+        {
+          type: 'text',
```
Comment (a-wong-8) [inline app/i18n/en/footer.ts:66]: this needs to be 'link'

Comment (gevuong) [reply app/i18n/en/footer.ts]: nice catch. Resolved in most recent commit.

**Review by a-wong-8 — APPROVED** [review]
excellent work

**Comment by gevuong** [comment]
@a-wong-8 thank you

**Pull request merged**